kimdaeun/vite-project/src/App.tsx

```diff
@@ -0,0 +1,11 @@
+import Counter from "./components/Counter.tsx";
+
+function App() {
+  return (
+    <>
+      <Counter></Counter>
+    </>
```

Comment:
p5:이렇게 수정하면 어떨까 합니다 :)
```suggestion
    <Counter/>
```